app/webpacker/components/RegistrationsV2/RegistrationEdit/Refunds.jsx

```diff
@@ -75,6 +77,21 @@ function RefundRow({
   refund, refundMutation, isMutating, userId, competitionId,
 }) {
   const [amountToRefund, setAmountToRefund] = useInputState(refund.ruby_amount_refundable);
+  const confirm = useConfirm();
+
+  const attemptRefund = () => confirm({
+    content: I18n.t('registrations.refund_confirmation'),
+  }).then(() => {
+    refundMutation({
+      competitionId,
+      userId,
+      paymentId: refund.payment_id,
+      paymentProvider: refund.payment_provider,
+      amount: amountToRefund,
+    });
+  }).catch(() => {
+
+  });
```

Comment:
Why is there an empty catch here?

Reply:
I thought useConfirm always needed it, but I guess not